.gitignore

```diff
@@ -12,5 +12,10 @@
 *.out
 
 # Dependency directories (remove the comment below to include it)
-bin
-.github
+#vendor/
```

Comment:
Can you undo this so it's unmodified from upstream? You can `git add -f` to add vendor files.